### ISSUE TITLE
Document update conflict received after upgrade from v1.0.112 to v1.0.142

### ISSUE THREAD

**Issue by lsoica** [Budibase/budibase#5728]
**Hosting**
- Self
  - Method: k8s
  - Budibase Version: v1.0.142

**Describe the bug**
"Document update conflict." Error received in both the designer and the published app for some of the pages:
<img width="591" alt="image" src="https://user-images.githubusercontent.com/1623282/166940722-67536d31-52c1-4dfb-b207-62c11d2c24f0.png">

Console log:
<img width="717" alt="image" src="https://user-images.githubusercontent.com/1623282/166941053-01770fb1-6c3e-4dbf-bccb-ce95506d0551.png">

App service container logs:
```
Error updating usage quotas for queries {
  error: 'conflict',
  reason: 'Document update conflict.',
  status: 409,
  name: 'conflict',
  message: 'Document update conflict.',
  stack: 'Error\n' +
    '    at generateErrorFromResponse (/app/node_modules/pouchdb/lib/index.js:694:18)\n' +
    '    at /app/node_modules/pouchdb/lib/index.js:6816:19\n' +
    '    at runMicrotasks (<anonymous>)\n' +
    '    at processTicksAndRejections (internal/process/task_queues.js:95:5)',
  docId: 'usage_quota'
}
ERROR [1651759676563] (28 on app-service-6c6bbd87b4-khz4l): Document update conflict.
    Error
        at generateErrorFromResponse (/app/node_modules/pouchdb/lib/index.js:694:18)
        at /app/node_modules/pouchdb/lib/index.js:6816:19
        at runMicrotasks (<anonymous>)
        at processTicksAndRejections (internal/process/task_queues.js:95:5)
Trace: Error
    at generateErrorFromResponse (/app/node_modules/pouchdb/lib/index.js:694:18)
    at /app/node_modules/pouchdb/lib/index.js:6816:19
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  message: 'Document update conflict.',
  error: 'conflict',
  reason: 'Document update conflict.',
  docId: 'usage_quota'
}
    at /app/dist/api/index.js:72:21
    at Generator.throw (<anonymous>)
    at rejected (/app/dist/api/index.js:6:65)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
INFO  [1651759676565] (28 on app-service-6c6bbd87b4-khz4l): request completed
    res: {
      "statusCode": 400,
      "headers": {
        "vary": "Accept-Encoding",
        "content-type": "application/json; charset=utf-8",
        "content-length": "52"
      }
    }
    responseTime: 596
    req: {
      "id": 298,
      "method": "POST",
      "url": "/api/v2/queries/query_datasource_plus_b71a9881bd32477c983ad226ede8171f_9c8d232e32e942b3b3cfe56da6d2c872",
      "headers": {
        "connection": "upgrade",
        "x-real-ip": "10.244.0.0",
        "x-forwarded-for": "10.241.124.175, 10.244.0.0",
        "host": "app-service.budibase.svc.cluster.local:4002",
        "content-length": "17",
        "x-request-id": "a576260e5608b803586d80cb0f085164",
        "x-forwarded-host": "<my domain here>",
        "x-forwarded-port": "443",
        "x-forwarded-proto": "https",
        "x-forwarded-scheme": "https",
        "x-scheme": "https",
        "accept": "application/json",
        "x-budibase-app-id": "app_default_813f6a34f90e4c9da9f7a22ef7f1634a",
        "x-csrf-token": "a2aeb12c-4e3b-4de4-be76-6935f9992c92",
        "x-budibase-api-version": "1",
        "x-budibase-type": "client",
        "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36",
        "content-type": "application/json",
        "sec-gpc": "1",
        "origin": "https://<my domain here>",
        "sec-fetch-site": "same-origin",
        "sec-fetch-mode": "cors",
        "sec-fetch-dest": "empty",
        "referer": "https://<my domain here>/app/<my app here>",
        "accept-encoding": "gzip, deflate, br",
        "accept-language": "en-GB,en-US;q=0.9,en;q=0.8",
        "cookie": "budibase:auth=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJ1c18zZWFiMzkyODAwYTk0ODY0ODczODg0YjM0MDc1OGFhYyIsInNlc3Npb25JZCI6IjRhYmRhYWVjY2NhODRiNzk5NTliNzdlMDk4YjI3MTExIiwidGVuYW50SWQiOiJkZWZhdWx0IiwiaWF0IjoxNjUxNzU4OTQwfQ.Fq_7h8o-nrYjagxoKEKdFhnw0l2TST8GdHyKMcCKlqE; budibase:auth.sig=wMeUzoD3S3zGZ-nhXPq3uXFKNHo; budibase:currentapp=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhcHBJZCI6ImFwcF9kZWZhdWx0XzgxM2Y2YTM0ZjkwZTRjOWRhOWY3YTIyZWY3ZjE2MzRhIiwiaWF0IjoxNjUxNzU5MDUzfQ.ABvn7YHxAob-2pNl7QvC7KwFP4vD5W2HylZy2f6uH8M"
      },
      "remoteAddress": "::ffff:10.244.1.200",
      "remotePort": 54042
    }

```

**Comment by Rory-Powell** [comment]
Hi @lsoica thanks for raising this. Looks like the high volume of queries being fired at once when your page loads is causing an issue with how we track the number of queries executed. I will try to replicate and come back with a fix.

**Comment by bjornclauw** [comment]
Hosting

- Self
   - Method: Docker
   - Budibase Version: v1.0.142

confirmed, same issue (5 api queries on pageload). App version 1.0.110 continues with above error. App version 1.0.142 halts query executions.

**Comment by oswaldB** [comment]
exactly same issue

**Comment by Rory-Powell** [comment]
Fix has been released in `1.0.144`

**Comment by oswaldB** [comment]
It is working fine for me.

Thank you so much :)
